interfaces/builtin/browser_support.go

```diff
@@ -166,6 +166,8 @@ owner @{PROC}/@{pid}/fd/[0-9]* w,
 /sys/devices/**/serial r,
 /sys/devices/**/vendor r,
 /sys/devices/system/node/node[0-9]*/meminfo r,
+/sys/devices/virtual/dmi/id/product_name r,
```

Comment:
If like me, you're wondering why chromium needs it, it's this commit that added it: https://chromium.googlesource.com/chromium/src.git/+/84618eee98fdf7548905e883e63e4f693800fcfa

Reply:
Thx, I'll update commit description.